IBAnimatable/AnimatableButton.swift

```diff
@@ -34,9 +34,23 @@ import UIKit
   }
   
   // MARK: - BorderDesignable
-  @IBInspectable public var borderColor: UIColor?
-  @IBInspectable public var borderWidth: CGFloat = CGFloat.NaN
-  @IBInspectable public var borderSide: String?
+  @IBInspectable public var borderColor: UIColor? {
+    didSet {
+      configBorder()
+    }
+  }
+  
+  @IBInspectable public var borderWidth: CGFloat = CGFloat.NaN {
+    didSet {
+      configBorder()
+    }
+  }
+  
+  @IBInspectable public var borderSide: String? {
+    didSet {
+      configBorder()
+    }
+  }
```

Comment:
If I remember correctly, I started with that and wanted to avoid calling `configBorder` multiple times, and move them out in `configInspectableProperties`. But should be that way for changing those properties after `awakeFromNib`.